pcli/src/network.rs

```diff
@@ -55,6 +63,20 @@ impl Opt {
 
         if code == 0 {
             println!("transaction submitted successfully");
+
+            if let Some(note_commitment) = await_change {
+                let mut view_client = self.view_client(fvk, self.data_path.as_ref()).await?;
+                tracing::debug!("awaiting change note");
+                view_client
+                    .note_by_commitment(NoteByCommitmentRequest {
+                        fvk_hash: Some(fvk.hash().into()),
+                        note_commitment: Some(note_commitment.into()),
+                        await_detection: true,
+                    })
```

Comment:
This forces the client to manually construct the proto type; the `ViewClient` should handle conversions from domain types.